ContextDesktop/src/main/java/com/github/migi_1/Context/model/entity/Commander.java

```diff
@@ -19,7 +19,7 @@
 
     //String of the path to the commander model
     private static final String PATHNAME = "Models/ninja.j3o";
-    private static final Vector3f MOVE_VECTOR = new Vector3f(-0.2f, 0, 0);
+    private static final Vector3f MOVE_VECTOR = new Vector3f(-0.5f, 0, 0);
 
```

Comment:
Forgot to change that back, changed it earlier for testing purposes. 
I will close the PR to fix this.